package.json

```diff
@@ -28,11 +28,17 @@
   },
   "homepage": "https://github.com/lob/hapi-bookshelf-total-count#readme",
   "devDependencies": {
```

Comment:
I'm not sure if im just missing it, but shouldn't bluebird be a normal dep?

also, since it assumes `hapi-query-filter`, `hapi`, and `bookshelf`, should they be peer deps?